### PR TITLE
DEV: Add spec to ensure banned phrases do not occur in en translations

### PR DESCRIPTION
**Pull request by davidtaylorhq**

### CODE REVIEW

**Comment by gschlager** [comment]
Are we running this spec for plugins or only core? I would have chosen `i18n_lint.rb` to implement these checks, but rspec is fine as well.

**Comment by davidtaylorhq** [comment]
Oh nice - thanks @gschlager. I'll move it there :+1:

**Comment by gschlager** [comment]
> Oh nice - thanks @gschlager. I'll move it there 👍

I gave it a try. :wink:  Quick hack: https://github.com/discourse/discourse/compare/i18n/banned-lint

**Review by gschlager — APPROVED** [review]

File: script/i18n_lint.rb [hunk 1]
```diff
@@ -100,6 +109,10 @@ def validate_content(yaml)
            !EXEMPTED_DOUBLE_CURLY_BRACKET_KEYS.include?(key)
         @errors[:invalid_interpolation_key_format] << key
       end
+
+      BANNED_PHRASES.keys.each do |banned|
+        @errors["banned_phrase_#{banned}"] << key if value.include?(banned)
```
Comment (gschlager) [inline script/i18n_lint.rb:114]: Oops, I missed the `downcase` from your implementation :blush: 
It should have been `value.downcase.include?(banned)`

Comment (davidtaylorhq) [reply script/i18n_lint.rb:114]: Ah yes, I'll do a followup 👀

**Pull request merged**